sound/soc/sof/intel/hda-dsp.c

```diff
@@ -688,6 +688,36 @@ static int hda_resume(struct snd_sof_dev *sdev, bool runtime_resume)
 	return 0;
 }
 
+/* Trigger streams which are ignored / stay active during suspend.
```

Comment:
the commit message is too convoluted.

"The current implementation does not power down links, stop command IO (CORB/RIRB) and stop the DMA streams, which on some platforms prevents actual use of S0ix. This patch modifies the suspend sequence to correct these three points, and applies a symmetric change for the resume sequence"